tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "lib": ["ESNext", "ESNext.AsyncIterable", "DOM"],
+    "jsx": "react",
```

Comment:
What's this for?

Reply:
Am I missing something to setup? Cause I don't have this error

![Capture d’écran 2022-06-30 à 3 16 13 PM](https://user-images.githubusercontent.com/9987732/176686761-f37d2021-7cbc-4699-8404-a5639e8b8dea.png)

Reply:
That's weird it's not even jsx, well let's keep it that way

Reply:
OK